packages/dialtone-css/lib/build/less/components/stack.less

```diff
@@ -84,47 +114,53 @@
 }
 
 //  ============================================================================
-//  $   RESPONSIVE
+//  $   SIZES
 //  ----------------------------------------------------------------------------
-//  $$  EXTRA LARGE
+.d-stack--gap-100 { --stack-gap: var(--dt-space-100); }
+.d-stack--gap-200 { --stack-gap: var(--dt-space-200); }
+.d-stack--gap-300 { --stack-gap: var(--dt-space-300); }
+.d-stack--gap-400 { --stack-gap: var(--dt-space-400); }
+.d-stack--gap-450 { --stack-gap: var(--dt-space-450); }
+.d-stack--gap-500 { --stack-gap: var(--dt-space-500); }
+.d-stack--gap-600 { --stack-gap: var(--dt-space-600); }
+
+
+//  ============================================================================
+//  $   RESPONSIVE
 //  ----------------------------------------------------------------------------
-.d-stack--xl {
-  @media screen and (max-width: 1264px) {
-    .direction-options();
-  }
-}
 
-//  $$  LARGE
+//  $$ SMALL
 //  ----------------------------------------------------------------------------
-.d-stack--lg {
-  @media screen and (max-width: 980px) {
+.d-stack--sm {
+  @media screen and (min-width: 0) and (max-width: 480px) {
     .direction-options();
+    .gap-options();
   }
 }
 
 //  $$  MEDIUM
 //  ----------------
 .d-stack--md {
-  @media screen and (max-width: 640px) {
+  @media screen and (min-width: 480px) and (max-width: 640px) {
     .direction-options();
+    .gap-options();
   }
 }
 
-//  $$ SMALL
+//  $$  LARGE
 //  ----------------------------------------------------------------------------
-.d-stack--sm {
-  @media screen and (max-width: 480px) {
+.d-stack--lg {
+  @media screen and (min-width: 640px) and (max-width: 980px) {
     .direction-options();
+    .gap-options();
   }
 }
```

Comment:
This isn't quite how screen mq should work. While yes, recognizing it's backwards, the range you've created with `min-width` and `max-width` limits it apply **only** within those bounds. That's against the grain of RWD. 

For example...
```less
.d-stack--lg {
  @media screen and (max-width: 980px) {
    ...
  }
}
```
... means "up to large size".